build-tools/automation/azure-pipelines.yaml

```diff
@@ -431,6 +431,17 @@ stages:
         artifactName: Xamarin.Android.JcwGen_Tests-Signed.apk
         artifactFolder: Default
 
+    - template: yaml-templates/apk-instrumentation.yaml
+      condition: and(succeeded(), eq(variables['XA.Commercial.Build'], 'true'))
```

Comment:
It seems like the `condition` here has to go under `parameters`? Weird.

![image](https://user-images.githubusercontent.com/840039/69089491-25e7f580-0a0e-11ea-8fd9-2fc7171afcfd.png)

My [VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azure-devops.azure-pipelines) complains about this, too.